src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom'
+import './App.css';
+import Transactions from './components/Transactions'
+import Operations from './components/Operations'
+import Breakdown from './components/Breakdown'
+import axios from 'axios'
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
```

Comment:
if this will be your big react fullstack project, I would suggest changing these components to functional components